docs/_docs/permalinks.md

```diff
@@ -4,27 +4,55 @@ title: Permalinks
 permalink: /docs/permalinks/
 ---
 
-Jekyll supports a flexible way to build your site’s URLs. You can specify the
-permalinks for your site through the [Configuration](../configuration/) or in
-the [YAML Front Matter](../frontmatter/) for each post. You’re free to choose
-one of the built-in styles to create your links or craft your own. The default
-style is `date`.
-
-Permalinks are constructed by creating a template URL where dynamic elements
-are represented by colon-prefixed keywords. For example, the default `date`
-permalink is defined according to the format `/:categories/:year/:month/:day/:title.html`.
-
-<div class="note info">
-  <h5>Specifying permalinks through the YAML Front Matter</h5>
```

Comment:
Actually, I'm not sure what this note even means. You can't add `permalink: /:categories/:year/:month/:day/:title.html` in your page's front matter or in your config's defaults. I took this back out again until I can clarify what it's saying. There would be no reason to actually add a permalink property in a single page anyway instead of in the configuration file.